helm/archive-node/templates/archive.yaml

```diff
@@ -109,7 +109,7 @@ spec:
         - name: postgres-port
           protocol: TCP
           containerPort: {{ .Values.archive.ports.postgres }}
-{{- include "healthcheck.archive.allChecks" .Values | indent 8 }}
+{{- include "healthcheck.archive.allChecks" . | indent 8 }}
```

Comment:
note: increased scope to all for inclusion of Release, Chart etc built-in objects